bin/mozetl-databricks.py

```diff
@@ -5,9 +5,48 @@
 import os
 import json
 import logging
+from base64 import b64encode
+from textwrap import dedent
 
 
-def run_submit(args, instance="dbc-caf9527b-e073.cloud.databricks.com"):
+def generate_runner(module_name, instance, token):
+    """Generate a runner for the current module to be run in Databricks."""
+
+    runner_data = """
+    # This runner has been auto-generated from mozilla/python_mozetl/bin/mozetl-databricks.py.
+    # Any changes made to the runner file will be over-written on subsequent runs.
+    from {module} import cli
+    
+    try:
+        cli.entry_point(auto_envvar_prefix="MOZETL")
+    except SystemExit:
+        # avoid calling sys.exit() in databricks
+        # http://click.palletsprojects.com/en/7.x/api/?highlight=auto_envvar_prefix#click.BaseCommand.main
+        pass
+    """.format(
+        module=module_name
+    )
+    logging.debug(dedent(runner_data))
+
+    request = {
+        "contents": b64encode(dedent(runner_data)),
```

Comment:
I think this should be `.decode('ascii')`-ed, so that json.dumps works below (it doesn't take bytes).

Also, in python 3 at least, I need to `.encode('utf-8')` before passing to `b64encode` (it doesn't want a string). So all in all, `b64encode(dedent(runner_data).encode('utf-8')).decode('ascii')`

Reply:
The script is running via python 2 (for compat reasons), so this isn't an issue there.

Reply:
It might be nice to make this forward compatible with python 3, if it's easy to do.